docs/conf.py

```diff
@@ -237,7 +237,7 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = project + "doc"
 
-# A dictionary of values to pass into the template engine’s context for all pages.
+# A dictionary of values to pass into the template engine`s context for all pages.
```

Comment:
Why this change?

Reply:
B/c I accidentally deleted the ignore for RUF00-something and it corrected this. I can revert... or we could just keep it 🤞.

Reply:
A backtick is just wrong - but the UTF-8 tick is also silly, so maybe you can replace by a regular `'`?

Reply:
Done!